site/docs/axis.md

```diff
@@ -80,12 +80,12 @@ To provide themes for all axes, the axis config `config: {axis: {...}}` can cont
 
 ### Labels
 
-{% include table.html props="shortTimeLabels" source="AxisConfig" %}
+{% include table.html props="labels,labelAngle,labelPadding,shortTimeLabels" source="AxisConfig" %}
 
 ### Ticks
 
-{% include table.html props="tickColor,tickRound,labelColor,labelFont,labelFontSize,labelLimit,tickWidth" source="AxisConfig" %}
+{% include table.html props="ticks,tickColor,tickExtra,tickRound,tickSize,labelColor,labelFont,labelFontSize,labelLimit,tickWidth" source="AxisConfig" %}
 
 ### Title
 
-{% include table.html props="titleBaseline,titleColor,titleFont,titleLimit,titleFontWeight,titleFontSize,titlePadding,titleMaxLength" source="AxisConfig" %}
+{% include table.html props="maxExtent,minExtent,titleAlign,titleAngle,titleBaseline,titleColor,titleFont,titleLimit,titleFontWeight,titleFontSize,titlePadding,titleMaxLength" source="AxisConfig" %}
```

Comment:
maxExtent,minExtent,titleAlign,titleAngle are in VgAxisBase so they should be in Axis Properties too (in addition to AxisConfig)?